resources/nats/values.yaml

```diff
@@ -0,0 +1,323 @@
+####################################
+#                                  #
+#       Global Configuration       #
+#                                  #
+####################################
+
+global:
+  containerRegistry:
+    path: eu.gcr.io/kyma-project
+  images:
+    nats:
+      name: nats
+      version: v20230314-2.9.15-alpine3.17
+      directory: external
+    nats_config_reloader:
+      name: natsio/nats-server-config-reloader
+      version: 0.10.1
+      directory: external
+    prometheus_nats_exporter:
+      name: natsio/prometheus-nats-exporter
+      version: 0.10.1
+      directory: external
+
+  jetstream:
+    # Storage type of the stream, memory or file.
+    storage: file
+    fileStorage:
+      size: 1Gi
+    # podManagementPolicy controls how pods are created during initial scale up,
+    # when replacing pods on nodes, or when scaling down.
+    podManagementPolicy: Parallel
+
+####################################
+#                                  #
+#  Security Context Configuration  #
+#                                  #
+####################################
+podSecurityContext:
+  fsGroup: 10001
+  seccompProfile:
+    type: RuntimeDefault
+
+containerSecurityContext:
+  privileged: false
+  allowPrivilegeEscalation: false
+  capabilities:
+    drop:
+      - ALL
+
+###############################
+#                             #
+#     Istio Configuration     #
+#                             #
+###############################
+istio:
+  enabled: false
+
+###############################
+#                             #
+#  NATS Server Configuration  #
+#                             #
+###############################
+nats:
```

Comment:
Does NATS upstream use this in their charts too?

Reply:
yes, its mentioned [here](https://github.com/nats-io/k8s/blob/fc718ec58e8dcf2334effa518388dd8ba0b6a3ce/helm/charts/nats/values.yaml#L6).